Public/Src/Engine/UnitTests/Scheduler/FileMonitoringViolationAnalyzerTests.cs

```diff
@@ -125,7 +132,9 @@ public void DoubleWriteEventLoggedOnViolationError()
                 exclusiveOpaqueDirectoryContent: null,
                 sharedOpaqueDirectoryWriteAccesses: null,
                 allowedUndeclaredReads: null,
-                absentPathProbesUnderOutputDirectories: null);
+                absentPathProbesUnderOutputDirectories: null,
+                ReadOnlyArray<(FileArtifact, FileMaterializationInfo, PipOutputOrigin)>.Empty,
+                out var _);

```

Comment:
>out var _ [](start = 16, length = 9)

nit: Can just be 'out _' (many of these)

Reply:
Yeap. Keep forgetting about that.

---
In reply to: [279903344](https://github.com/Microsoft/BuildXL/pull/224#discussion_r279903344) [](ancestors = 279903344)